.meshrc.yaml

```diff
@@ -206,9 +220,9 @@ additionalTypeDefs: |
   }
 additionalResolvers:
   - ./resolvers.ts
-plugins:
-  - csrfPrevention:
-      requestHeaders: ['x-graphql-yoga-csrf']
+# plugins:
+#   - csrfPrevention:
+#       requestHeaders: ['x-graphql-yoga-csrf']
```

Comment:
I know this PR is WIP, but we do need to enable this before we can merge the PR.

Reply:
bump on this, we need to uncomment this and make sure things work.